src/tests/ftest/launch.py

```diff
@@ -1016,6 +1038,70 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+def report_skipped_test(test_file, avocado_logs_dir, reason):
+    """Report an error for the skipped test.
+
+    Args:
+        test_file (str): the test python file
+        avocado_logs_dir (str): avocado job-results directory
+        reason (str): [description]
```

Comment:
```suggestion
        reason (str): [description]
```
Missing discription.

Reply:
Fixed, thanks.